keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -281,7 +281,7 @@ def __init__(
         self,
         vocabulary=None,
         sequence_length: int = None,
-        lowercase: bool = False,
```

Comment:
Lowercase=False is actually the default we want here! The incorrect default is above in the pretokenizer function.

Reply:
So, changes will be in pretokenizer function right?


https://github.com/ADITYADAS1999/keras-nlp/blob/my_keras_npl_repo/keras_nlp/tokenizers/word_piece_tokenizer.py#L96